tests/automation_framework/data/work_order/work_order_submit_without_verifyingkey.json

```diff
@@ -0,0 +1,36 @@
+{
```

Comment:
Remove this file in your next commit. We don't need the file since we are removing the test,